db.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
```

Comment:
are you using this file?

Reply:
no mam I just figured out this file was not used so I removed it.